src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -27,13 +27,13 @@
 namespace PrestaShopBundle\Controller\Admin;
 
 use Exception;
-use PrestaShop\PrestaShop\Adapter\Module\Module as ModuleAdapter;
 use PrestaShop\PrestaShop\Core\Addon\AddonListFilter;
 use PrestaShop\PrestaShop\Core\Addon\AddonListFilterStatus;
 use PrestaShop\PrestaShop\Core\Addon\AddonListFilterType;
 use PrestaShop\PrestaShop\Core\Addon\AddonsCollection;
 use PrestaShop\PrestaShop\Core\Addon\Module\ModuleRepository;
 use PrestaShop\PrestaShop\Core\Addon\Module\Exception\UnconfirmedModuleActionException;
+use PrestaShopBundle\Controller\Admin\Improve\Modules\AbstractController;
```

Comment:
I'm not really fan of `AbstractController` for 2 reasons:

* first, Symfony also have an AbstractController, at some point we will think we use the Symfony one
* second, this doesn't help to figure out why we use this controller

How about `AbstractModuleController` ?